src/Components/CreateImageWizard/utilities/generateDefaultName.ts

```diff
@@ -1,15 +1,12 @@
-import { ImageTypes } from '../../../store/imageBuilderApi';
-
-export const generateDefaultName = (
-  distribution: string,
-  arch: string,
-  targetEnvironments: ImageTypes[]
-) => {
+export const generateDefaultName = (distribution: string, arch: string) => {
```

Comment:
Ah, one more thing. We have proper types for those, let's use them:

```suggestion
export const generateDefaultName = (
  distribution: Distributions,
  arch: ImageRequest['architecture']
) => {
```